### PR TITLE
Nodejs version upgraded to 20.0.0

### PR DESCRIPTION
**Pull request by VenkataChalla554**
Nodejs version upgraded to 22.0.0.

Tested : https://github.com/DFE-Digital/apply-for-teacher-training/actions/runs/9018382199

### CODE REVIEW

File: .tool-versions [hunk 1]
```diff
@@ -1 +1 @@
-nodejs 16.17.0
+nodejs 22.0.0
```
Comment (RMcVelia) [inline .tool-versions:1]: 20 as we are using node20 in the action?

Comment (VenkataChalla554) [reply .tool-versions]: Fixed and test with test branch.

**Review by RMcVelia — APPROVED** [review]

**Comment by VenkataChalla554** [comment]
[update node for keyvault-yaml-secret](https://trello.com/c/fQSmELPm/1721-update-node-for-keyvault-yaml-secret)

**Comment by VenkataChalla554** [comment]
[update node for keyvault-yaml-secret](https://trello.com/c/fQSmELPm/1721-update-node-for-keyvault-yaml-secret)

**Pull request merged**